code/game/objects/structures/table_frames.dm

```diff
@@ -108,6 +108,6 @@
 				return
 			to_chat(user, "<span class='notice'>You start adding [material] to [src]...</span>")
 			if(do_after(user, 20, target = src) && material.use(1))
-				make_new_table(toConstruct, null, type)
+				make_new_table(toConstruct, null, null)
```

Comment:
you can just pass toConstruct, the rest will default to null on their own.

Also mind finding another way to achieve https://github.com/tgstation/tgstation/pull/54869? you should also just straight up remove buildstack as an arg, since it's only used by this line.

Reply:
Yeah, but I though null would make clearer that we are purposedly leaving that argument blank, for code clarity. But in the end it doesn't matter so whatever.
And ok, I will try tomorrow.